src/_includes/layouts/form-job-posting.liquid

```diff
@@ -21,15 +21,15 @@
                 <legend>I want to post a vacancy</legend>
                 <p class="form-element">
                 <label for="form-1-input-0" class="form-label">Contact person</label>
-                <input type="text" name="Contact person" id="form-1-input-0" class="text" required>
+                <input type="text" name="Contact person" id="form-1-input-0" class="text" autocomplete="name" required>
                 </p>
                 <p class="form-element">
                 <label for="form-1-input-1" class="form-label">Contact person's email address</label>
-                <input type="email" name="Contact person's email address" id="form-1-input-1" class="text" required>
+                <input type="email" name="Contact person's email address" id="form-1-input-1" class="text" autocomplete="email work" required>
                 </p>
                 <p class="form-element">
                 <label for="form-1-input-2" class="form-label">Phone number</label>
-                <input type="text" name="Phone number" id="form-1-input-2" class="text" required>
+                <input type="text" name="Phone number" id="form-1-input-2" class="text" autocomplete="tel work" required>
```

Comment:
Consider adding `section-contact`, ie. `section-contact name` and `section-contact email` to future proof this form in case we want to collect both public contact person and private submitter person.

All matching `section-xxx` fields will be filled in together and having `section-submitter` later allows you to autocomplete both in two-goes, instead of only one working or one filling in both.